android/app/src/main/java/org/fossasia/openevent/api/DataDownloadManager.java

```diff
@@ -29,28 +40,38 @@ public static DataDownloadManager getInstance() {
     }
 
     public void downloadEvents() {
-        client.getOpenEventAPI().getEvents().enqueue(new EventListResponseProcessor());
+        openEventClient.getOpenEventAPI().getEvents().enqueue(new EventListResponseProcessor());
     }
 
     public void downloadSpeakers() {
-        client.getOpenEventAPI().getSpeakers().enqueue(new SpeakerListResponseProcessor());
+        openEventClient.getOpenEventAPI().getSpeakers().enqueue(new SpeakerListResponseProcessor());
     }
 
     public void downloadSponsors() {
-        client.getOpenEventAPI().getSponsors().enqueue(new SponsorListResponseProcessor());
+        openEventClient.getOpenEventAPI().getSponsors().enqueue(new SponsorListResponseProcessor());
     }
 
     public void downloadSession() {
-        client.getOpenEventAPI().getSessions("start_time.asc").enqueue(new SessionListResponseProcessor());
+        openEventClient.getOpenEventAPI().getSessions("start_time.asc").enqueue(new SessionListResponseProcessor());
     }
 
     public void downloadTracks() {
-        client.getOpenEventAPI().getTracks().enqueue(new TrackListResponseProcessor());
+        openEventClient.getOpenEventAPI().getTracks().enqueue(new TrackListResponseProcessor());
     }
 
     public void downloadMicrolocations() {
-        client.getOpenEventAPI().getMicrolocations().enqueue(new MicrolocationListResponseProcessor());
+        openEventClient.getOpenEventAPI().getMicrolocations().enqueue(new MicrolocationListResponseProcessor());
     }
 
+    public void downloadPageId(Context context, String eventName, String accessToken) {
```

Comment:
Though I still think the download calls should be included in the data download manager and it should be subscribed wherever necessary to prevent code redundancy (I will be observing pageid at two different classes - Main activity and feed fragment. Views @iamareebjamal?